docs/admin/kubelet-tls-bootstrapping.md

```diff
@@ -92,38 +92,43 @@ tracked at [kubernetes/features#267](https://github.com/kubernetes/features/issu
 --feature-gates=RotateKubeletServerCertificate=true
 ```
 
-The following RBAC `ClusterRoles` represent the `nodeclient`, `selfnodeclient`, and `selfnodeserver` capabilities. Similar roles
-may be automatically created in future releases.
+The following RBAC `ClusterRoles` represent the `nodeclient`, `selfnodeclient`, and `selfnodeserver` capabilities.
+Some of these roles are automatically created since the v1.8.0 release.
+Prior to that, you had to create these resources yourself.
 
 ```yml
 # A ClusterRole which instructs the CSR approver to approve a user requesting
 # node client credentials.
+# This ClusterRole is automatically created since v1.8.0
 kind: ClusterRole
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
-  name: approve-node-client-csr
+  name: system:certificates.k8s.io:certificatesigningrequests:nodeclient
```

Comment:
I wouldn't give examples which encourage people to create things with system-prefixed names

Reply:
I guess this should now mostly tell the user what's there, right?

Reply:
How can I rephrase this to be "look what's created for you"?

Reply:
Maybe just remove the cluster roles entirely and add something like

> The "...nodeclient", "...selfnodeclient", and "...selfnodeserver" allow uses to be approved for CRS where... (describe what these roles allow)
> To grant them to a user, such as a bootstrapping token, and admin should create the following `ClusterRoleBindings`:

Reply:
Is as-is ok for now? Going to bed now...